cloud-nio/cloud-nio-impl-ftp/src/main/scala/cloud/nio/impl/ftp/FtpClientPool.scala

```diff
@@ -1,22 +1,19 @@
 package cloud.nio.impl.ftp
 
-import cloud.nio.impl.ftp.FtpUtil.FtpOperation
 import com.typesafe.scalalogging.StrictLogging
 import io.github.andrebeat.pool.{ExpiringPool, ReferenceType}
 import org.apache.commons.net.ftp.FTPClient
 
 import scala.concurrent.duration._
 
 object FtpClientPool extends StrictLogging {
-  case class DataConnectionToken(operation: FtpOperation)
-  
   def dispose(ftpClient: FTPClient) = try {
     if (ftpClient.isConnected) {
       ftpClient.logout()
       ftpClient.disconnect()
     }
   } catch {
-    case e: Exception => logger.error("Failed to disconnect ftp client", e)
+    case e: Exception => logger.debug("Failed to disconnect ftp client", e)
```

Comment:
TOL that's jumping quite a few levels of severity

Reply:
I agree but it turns out it happens a lot. I didn't dig much further as to why but at this point there's nothing we can really do about it and it's not that critical since we'll recreate a connection after that. I don't think I'll have time to clear this up before the demo next week but I'll make a ticket for later.